conf/application.conf

```diff
@@ -265,6 +266,7 @@ upliftFailure {
 feature-switch {
   comparablePropertiesEnabled = true
   draftListEnabled = false
+  agentJourney2026Enabled = true
```

Comment:
Please can you set this to false by default. Could you also please create some prs for app-config to have this disabled in Prod and QA in case somebody accidentally changes this to true